### PR TITLE
test and fix green kubo diffusion coefficient

### LINKED ISSUE #381: Fix gk self diffusion test
After #380 the GK self diffusion test was disabled. The test has to be fixed asap.

### LINKED ISSUE #509: VACF wrong in GK diffusion calculator
The values for the autocorrelation have the wrong scale and unit

### PR DESCRIPTION
**Pull request by christophlohrmann**
fix #509
fix #381

### CODE REVIEW

File: CI/integration_tests/calculators/_test_green_kubo_self_diffusion_coefficients.py [hunk 1]
```diff
@@ -26,39 +26,84 @@
...
+    exp.add_data(proc)
+
+    res = exp.run.GreenKuboDiffusionCoefficients(data_range=vacf_range, plot=False)[
+        species.name
+    ]
+
+    time_should_be = time_step * np.arange(0, vacf_range) * units.time
+    thermal_vel_SI = np.sqrt(3 * kT / mass) * units.length / units.time
+    relaxation_time_SI = relaxation_time * units.time
+    vacf_should_be = thermal_vel_SI ** 2 * np.exp(-time_should_be / relaxation_time_SI)
```
Comment (SamTov) [inline CI/integration_tests/calculators/_test_green_kubo_self_diffusion_coefficients.py:102]: nice

**Review by SamTov — APPROVED** [review]
I like it. In principle we could use an underscore and remove the full test rather than comment out the test parts but I am happy either way as it should be restored very soon.

**Comment by christophlohrmann** [comment]
@PythonFZ do you know what is going on with CI here?

**Comment by PythonFZ** [comment]
> @PythonFZ do you know what is going on with CI here?

Probably some issue with the change of base. Triggering a new CI run seems to have fixed it.

**Pull request merged**